app/views/uploaded_files/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p id="notice"><%= notice %></p>
```

Comment:
This is already handled in the wrapping application html. 

- [ ] Please remove it